presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -120,12 +116,7 @@ public class FeaturesConfig
     private boolean pushTableWriteThroughUnion = true;
     private boolean exchangeCompressionEnabled;
     private boolean exchangeChecksumEnabled;
-    private boolean legacyArrayAgg;
-    private boolean reduceAggForComplexTypesEnabled = true;
-    private boolean legacyLogFunction;
-    private boolean useAlternativeFunctionSignatures;
     private boolean legacyTimestamp = true;
```

Comment:
what about legacyTimestamp, legacyRowFieldOrdinalAccess and legacyCharToVarcharCoercion?

Reply:
also fieldNameInJsonCastEnabled and legacyJsonCast

Reply:
I've added all the session properties that delegate to system properties in `FeaturesConfig` in `Session#getSqlFunctionProperties`, which includes all of the above, and also `legacyCharToVarcharCoercion`.

Reply:
Also `legacyJsonCast`.  All have been added.